spec/peeps_spec.rb

```diff
@@ -0,0 +1,33 @@
+
+require 'peeps'
+
+describe Peep do
+  describe '#self.all' do
+    it 'returns all peeps' do
+      connection = PG.connect(dbname: 'chitter_test')
```

Comment:
Is there a point to this line if in our peep.rb we have an if statement to connect to the chitter_test db when rspec is run.